core/config/docs/docs_test.go

```diff
@@ -82,8 +82,14 @@ func TestDoc(t *testing.T) {
 		docDefaults.OperatorFactoryAddress = nil
 		require.Empty(t, docDefaults.ChainWriter.FromAddress)
 		require.Empty(t, docDefaults.ChainWriter.ForwarderAddress)
+		require.Zero(t, docDefaults.ChainWriter.GasLimit)
+		require.Equal(t, "", docDefaults.ChainWriter.Checker)
```

Comment:
Do these ones belong here?

Reply:
I don't know much about the context of this file. I was just mirroring what Blaz did originally when he setup the write capability and the ChainWriter config for keystone.

Reply:
This is an exceptional list of nilable fields w/o defaults only.